Resources/Prototypes/Recipes/Reactions/drinks.yml

```diff
@@ -98,9 +98,9 @@
     Tequila:
       amount: 1
     VodkaRedBool:
-      amount: 2
+      amount: 3
   products:
-    BudgetInsulsDrink: 3
+    BudgetInsulsDrink: 4
```

Comment:
This is probably fine in this instance, but I would generally advise splitting the changes you've made into multiple prs for the future:
1. Flavor/Description/localization changes.
2. Recipe changes.
3. Priority change for ejecting from all dispensers.

It makes hard for people to decide if they like the individual changes that you have made (localization & recipe changes) as you've kinda forced it as a "bundle deal." This can make it harder to review and get approved.

Relevant section of the PR Guidelines:
> Try to split your PR into smaller ones where it makes sense to do so. This makes it significantly easier to read and can lead to faster reviews. It’s also usually easier for you, and means you will receive earlier feedback and can avoid spending time making changes that have to be reworked.

https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html#content

Reply:
Sorry, I won't bundle everything together again.

Reply:
Don't sweat it, it's not that big of a deal. It's more of a stylistic choice.